### ISSUE TITLE
Remove android.support dependencies from teads Android project 

### ISSUE THREAD

**Issue by mr-procrastinator** [teads/TeadsSDK-android#163]
We could not drop Jetifer from our project due to the teads SDK. 
Do you really need android.support dep? 
![SS-05-072021 at 09 25 32](https://user-images.githubusercontent.com/4267524/117406983-90e4ff00-af16-11eb-8b19-dd4a13e2c812.png)
Check your SDK code with https://github.com/plnice/can-i-drop-jetifier

**Comment by benjaminvolland** [comment]
Hello @mr-procrastinator,
We have planned to migrate to androidX in a future version, for early June. 
It should fix your issue.

**Comment by benjaminvolland** [comment]
The support of androidX is now available in the last release:
https://github.com/teads/TeadsSDK-android/releases/tag/v4.9.0

**Comment by gbero** [comment]
I'm still getting the following warning though : 

`[Processor] Library '/Users/myuser/.gradle/caches/modules-2/files-2.1/tv.teads.sdk.android/sdk/4.9.1/24096fab0617b937710f9bae94745e9b7add4d3c/sdk-4.9.1.aar' contains references to both AndroidX and old support library. This seems like the library is partially migrated. Jetifier will try to rewrite the library anyway.`

**Comment by github-benjamin-volland** [comment]
Hi @gbero , thanks for the warn. I will fix it for the next release.

**Comment by herrbert74** [comment]
@github-benjamin-volland
This is still not fixed in 5.0.10.
Not only do you have to change to androidX, but you also have to update your dependencies.
You still depend on picasso:2.71828 from 2018, tv.teads.exoplayer:exoplayer-core:r2.4.3, and tv.teads.visdroid:library:1:10, which in turn depend on the support libraries.
Also, if you remove jetifier from gradle.properties, you should have already seen these warnings, which proves that this issue is actually not fixed. 

On the other hand, it seems that excluding the above removes the warnings.

```
implementation(tv.teads.sdk.android:sdk:5.0.10@aar) {
        isTransitive = true
        exclude(group = "tv.teads.visdroid", module = "library")
        exclude(group = "com.squareup.picasso", module = "picasso")
        exclude(group = "tv.teads.exoplayer", module = "exoplayer-core")
    }
```
